pkg/watermark/fetch/edge_fetcher.go

```diff
@@ -198,7 +198,6 @@ func (e *edgeFetcher) updateHeadIdleWMB(fromPartitionIdx int32) wmb.WMB {
 
 // Close function stops the processor manager.
 func (e *edgeFetcher) Close() error {
-	e.processorManager.Stop()
```

Comment:
Who does it right now?

Reply:
Stop() function will execute at the point of its creation, which could be within the map_udf or reduce_udf functions.